src/Audio/Cue.cs

```diff
@@ -774,6 +783,14 @@ internal bool INTERNAL_update()
 								long elapsedFromStop = INTERNAL_timer.ElapsedMilliseconds - INTERNAL_fadeStart;
 								variableValue = elapsedFromStop;
 							}
+							else
+							{
+								variableValue = 0.0f;
+							}
+						}
+						else if (curRPC.Variable.Equals("NumCueInstances"))
```

Comment:
At first I thought there was a missing SetVariable() somewhere, but when I went to plumb that in I ran into the public,private restrictions since it is a public API. For a bit I started factoring out private versions without those restrictions, but then found that NumCueInstances was being handled a bit differently, so just decided to keep things local here for now.